kvk/Issues/official/description.xml

```diff
@@ -49,54 +259,194 @@
+				<nl>De bijzondere rechtstoestand van de functionaris, zoals: Faillissement, schuldsanering of surseance van betaling</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="beperkingInRechtshandelingFunctionaris">
+			<Name>
+				<en>Restriction in legal action of the official</en>
+				<nl>Beperking in rechtshandeling van de functionaris</nl>
+			</Name>
+			<Description>
+				<en>The restriction in legal action of the official, zoals: Provisional rule, placed under guardianship of placed under administration</en>
+				<nl>De beperking in rechtshandeling van de functionaris, zoals: Provisioneel bewind, onder curatele gesteld of onder bewind gesteld</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="schorsingAanvang">
+			<Name>
+				<en>Start date of suspension</en>
+				<nl>Aanvangsdatum van de schorsing</nl>
+			</Name>
+			<Description>
+				<en>Indicates when the official's suspension started</en>
+				<nl>Geeft aan wanneer de schorsing van de functionaris is gestart</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="schorsingEinde">
+			<Name>
+				<en>End date of suspension</en>
+				<nl>Einddatum van de schorsing</nl>
+			</Name>
+			<Description>
+				<en>Indicates when the official's suspension has ended</en>
+				<nl>Geeft aan wanneer de schorsing van de functionaris beeindigd is</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="handlichting">
+			<Name>
+				<en>Emancipation of a minor</en>
+				<nl>Handlichting</nl>
+			</Name>
+			<Description>
+				<en>Emancipation of a minor</en>
+				<nl>Handlichting</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="heeftBeperking">
+			<Name>
+				<en>Has restrictions</en>
+				<nl>Heeft beperking</nl>
+			</Name>
+			<Description>
+				<en>Indicatie of er sprake is van enige vorm van beperking, die van invloed is op het bepalen van een bevoegdheid</en>
+				<nl>Indicatie of er sprake is van enige vorm van beperking, die van invloed is op het bepalen van een bevoegdheid</nl>
 			</Description>
 		</Attribute>
-		<Attribute id="kvkNumber">
+		<Attribute id="isBevoegd">
 			<Name>
-				<en>KVK number</en>
-				<nl>KVK nummer</nl>
+				<en>Is authorized</en>
+				<nl>Is bevoegd</nl>
 			</Name>
 			<Description>
-				<en>KVK number</en>
-				<nl>KVK nummer</nl>
+				<en>Indication of whether someone is authorized</en>
+				<nl>Indicatie of iemand bevoegd is</nl>
 			</Description>
 		</Attribute>
-		<Attribute id="sbiCode">
+		<Attribute id="reden">
 			<Name>
-				<en>SBI code</en>
-				<nl>SBI code</nl>
+				<en>Reason for established authority</en>
+				<nl>Reden van vastgestelde bevoegdheid</nl>
 			</Name>
 			<Description>
-				<en>SBI code</en>
-				<nl>SBI code</nl>
+				<en>The reason for (not) being authorized or not being able to determine someone's authority</en>
+				<nl>De reden voor het niet/wel bevoegd zijn of niet kunnen vastellen van iemands bevoegdheid</nl>
```

Comment:
In existing credentials the order of attributes cannot be changed. This will cause issues in the IRMA-apps where this credential is already present. You either have to extend the current credential by putting all new attributes at the end of the `<Attributes>` list and make them optional (such that apps that already have the credential will not get confused) or you have to start a new credential.

Reply:
I think no one has this credential yet

Reply:
It has been published in the IRMA app, so therefore users could have issued one to themselves. The app really doesn't handle this scenario well, so therefore we cannot allow it. Then that user experiences a potentially crashing IRMA app.

Reply:
Nice. This seems better. Can you add `optional="true"` to all attributes that you made new here in irma-demo? Then older credential instances keep working too. When we later add a equivalent in production (`pbdf`), we can omit the `optional="true"` again, since the credential doesn't exist yet there.

I.e.:
```
<Attribute id="referenceMoment" optional="true">
```

Reply:
Done

Reply:
Nice, this seems okay now.

Another merge came in between on `master` and we enforce a lineair history to prevent signing issues. Can you rebase to the current master and resign everything? Then everything should be fine.

Reply:
I rebased to the current master. If I do a 'irma scheme sign' 3 files are changed. Do I need to push those changes as well?

Reply:
Yes please. A clean checkout should say `Verification was successful.` when running `irma scheme verify`.

Reply:
Done